spark/src/main/scala/io/delta/tables/DeltaTable.scala

```diff
@@ -562,6 +562,203 @@ class DeltaTable private[tables](
           TableFeatureProtocolUtils.FEATURE_PROP_SUPPORTED))
     toDataset(sparkSession, alterTableCmd)
   }
+
+  /**
+   * Clone a DeltaTable to a given destination to mirror the existing table's data and metadata.
+   *
+   * Specifying properties here means that the target will override any properties with the same key
+   * in the source table with the user-defined properties.
+   *
+   * An example would be
+   * {{{
+   *  io.delta.tables.DeltaTable.clone(
+   *   "/some/path",
```

Comment:
Nit: Same for other "/some/path".

```suggestion
   *   "/some/path/to/table",
```